modules/log6x-rn-6.0.3.adoc

```diff
@@ -0,0 +1,43 @@
+
+[id="logging-release-notes-6-0-3-enhancements_{context}"]
+== New features and enhancements
+
+* With this update, the {loki-op} supports the configuring of the workload identity federation on the {gcp-first} by using the Cluster Credential Operator (CCO) in {product-title} 4.17 or later. (link:https://issues.redhat.com/browse/LOG-6421[LOG-6421])
+
+[id="logging-release-notes-6-0-3-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the collector used the default settings to collect audit logs, which did not account for back pressure from output receivers. With this update, the audit log collection is optimized for file handling and log reading. (link:https://issues.redhat.com/browse/LOG-6034[LOG-6034])
+
+* Before this update, any namespace containing `openshift` or `kube` was treated as an infrastructure namespace. With this update, only the following namespaces are treated as infrastructure namespaces: `default`, `kube`, `openshift`, and namespaces that begin with `openshift-` or `kube-`. (link:https://issues.redhat.com/browse/LOG-6204[LOG-6204])
+
+* Before this update, an input receiver service was repeatedly created and deleted, causing issues with mounting the TLS secrets. With this update, the service is created once and only deleted if it is not defined in the `ClusterLogForwarder` custom resource. (link:https://issues.redhat.com/browse/LOG-6343[LOG-6343])
+
+* Before this update, pipeline validation might enter an infinite loop if a name was a substring of another name. With this update, stricter name equality checks prevent the infinite loop. (link:https://issues.redhat.com/browse/LOG-6352[LOG-6352])
+
+* Before this update, the collector alerting rules included the summary and message fields. With this update, the collector alerting rules include the summary and description fields. (link:https://issues.redhat.com/browse/LOG-6406[LOG-6406])
+
+* Before this update, setting up the custom audit inputs in the `ClusterLogForwarder` custom resource with configured `LokiStack` output caused errors due to the nil pointer dereference. With this update, the Operator performs the nil checks, preventing such errors. (link:https://issues.redhat.com/browse/LOG-6441[LOG-6441])
+
+* Before this update, the collector did not correctly mount the `/var/log/oauth-server/` path, which prevented the collection of the audit logs. With this update, the volume mount is added, and the audit logs are collected as expected. (link:https://issues.redhat.com/browse/LOG-6486[LOG-6486])
+
+* Before this update, the collector did not correctly mount the `oauth-apiserver` audit log file. As a result, such audit logs were not collected. With this update, the volume mount is correctly mounted, and the logs are collected as expected. (link:https://issues.redhat.com/browse/LOG-6543[LOG-6543])
+
+[id="logging-release-notes-6-0-3-CVEs_{context}"]
+== CVEs
```

Comment:
@max-cx Below CVE's under errata are missing here
https://access.redhat.com/security/cve/CVE-2019-12900
https://access.redhat.com/security/cve/CVE-2024-10963
https://access.redhat.com/security/cve/CVE-2024-50602

I don't see https://access.redhat.com/security/cve/CVE-2024-1004 under 6.0.3

Errata: https://errata.devel.redhat.com/advisory/details/143291

Reply:
Yes, thank you for the heads up. I already updated the PR against the errata.